dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/entity/AuditLog.java

```diff
@@ -0,0 +1,75 @@
+package org.apache.dolphinscheduler.dao.entity;
+
+import com.baomidou.mybatisplus.annotation.IdType;
+import com.baomidou.mybatisplus.annotation.TableId;
+import com.baomidou.mybatisplus.annotation.TableName;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import org.apache.dolphinscheduler.common.enums.AuditModuleType;
+import org.apache.dolphinscheduler.common.enums.AuditOperationType;
+
+import java.util.Date;
+
+@TableName("t_ds_audit_log")
+public class AuditLog {
+
+    /**
+     * id
+     */
+    @TableId(value = "id", type = IdType.AUTO)
+    private int id;
+
+    /**
+     * user id
+     */
+    private int userId;
+
+    /**
+     * operation module
+     */
+    private AuditModuleType module;
```

Comment:
Don't use Enum here, this might make code comples.

Reply:
> Don't use Enum here, this might make code comples.

If so, user might need to use input boxes instead of drop-down boxes to query audit log. 
In this way, when inserting new audit logs, we need to input modules and operations in String class again.